pkg/util/time.go

```diff
@@ -98,3 +99,34 @@ func NewDisableableTicker(interval time.Duration) (func(), <-chan time.Time) {
 	tick := time.NewTicker(interval)
 	return func() { tick.Stop() }, tick.C
 }
+
+// JSONSecondsTimestamp is time.Time, but stored as Unix seconds integer value when marshalled to JSON.
+// Zero time.Time value is marshalled as 0.
+type JSONSecondsTimestamp time.Time
+
+func (t JSONSecondsTimestamp) MarshalJSON() ([]byte, error) {
+	if t.Time().IsZero() {
+		return []byte("0"), nil
+	}
+	return []byte(strconv.FormatInt(time.Time(t).Unix(), 10)), nil
+}
+
+func (t *JSONSecondsTimestamp) UnmarshalJSON(data []byte) error {
+	i, err := strconv.ParseInt(string(data[:]), 10, 64)
```

Comment:
Nit, but what do you gain by re-slicing `data`?

Reply:
Nothing really, copy&paste bug :(